src/Hl7.Fhir.Specification.Tests/Validation/BasicValidationTests.cs

```diff
@@ -1390,6 +1390,49 @@ public void ValidateWithTargetProfileAndChildDefinitions()
 
         }
 
+        [Fact]
+        public async void TestValidationPerformanceUsCore()
+        {
+            var resolver = new MultiResolver(new CachedResolver(new DirectorySource(Path.Combine("TestData", "us-core-profiles"))), new CachedResolver(new ZipSource("specification.zip")));
+
+            var ctx = new ValidationSettings()
+            {
+                ResourceResolver = resolver,
+                GenerateSnapshot = true,
+                EnableXsdValidation = false,
+                Trace = false,
+                ResolveExternalReferences = false
+            };
+
+            _validator = new Validator(ctx);
+
+            var instances = new[] { "us-core-instance-1.json", 
+                                    "us-core-instance-10.json", 
+                                    "us-core-instance-25.json", 
+                                    "us-core-instance-50.json",
+                                    "us-core-instance-100.json",
+                                    "us-core-instance-200.json",
+                                    "us-core-instance-300.json",
+                                    "us-core-instance-500.json",
+                                    "us-core-instance-700.json"};
+
+            foreach(var instance in instances)
+            {
+                var jsonBundle = await File.ReadAllTextAsync(Path.Combine("TestData", "us-core-instances", instance));
+                var parser = new FhirJsonParser();
+                var bundle = await parser.ParseAsync<Bundle>(jsonBundle);
+                Assert.NotNull(bundle);
+
+                var sw = new Stopwatch();
+                sw.Start();
+                var result = _validator.Validate(bundle);
+                sw.Stop();
```

Comment:
"When a [Stopwatch](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.stopwatch?view=net-6.0) instance measures more than one interval, the [Stop](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.stopwatch.stop?view=net-6.0) method is equivalent to pausing the elapsed time measurement. A subsequent call to [Start](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.stopwatch.start?view=net-6.0) resumes measuring time from the current elapsed time value. Use the [Reset](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.stopwatch.reset?view=net-6.0) method to clear the cumulative elapsed time in a [Stopwatch](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.stopwatch?view=net-6.0) instance."
https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.stopwatch.stop?view=net-6.0